ts/WoltLabSuite/Core/Component/GoogleMaps/Geocoding.ts

```diff
@@ -116,7 +116,7 @@ class Geocoding {
     this.#element.dataset.googleMapsLng = lng.toString();
 
     if (this.#element.hasAttribute("data-google-maps-geocoding-store") && this.#element.form) {
-      this.#store(this.#element.dataset.googleMapsGeocdingStore || "", lat, lng);
+      this.#store(this.#element.dataset.googleMapsGeocodingStore || "", lat, lng);
     }
```

Comment:
This whole logic looks a little odd anyway. Can you check whether the following works:
```
if ("googleMapsGeocodingStore" in this.#element.dataset && this.#element.form) {
  this.#store(this.#element.dataset.googleMapsGeocodingStore, lat, lng);
}
```
This should hopefully cleanly type check.